shotover-proxy/src/transforms/cassandra/sink_cluster/mod.rs

```diff
@@ -174,20 +173,47 @@ impl CassandraSinkCluster {
     }
 }
 
-fn create_query(messages: &Messages, query: &str, version: Version) -> Result<Message> {
+//Creating the AST is pretty expensive. Since we are defining the queries
+// ourselves and know they are not batch statements we can skip that check.
+// This function checks the messages list to ensure that the stream id used is not
+// already being used
+fn create_query_with_generated_stream_id(
```

Comment:
How about naming `create_query_with_unique_stream_id` otherwise its not clear what kind of generated value is used.